src/webhook/validation/deprecation.go

```diff
@@ -47,6 +64,14 @@ func deprecatedFeatureFlagDisableMetadataEnrichment(_ *dynakubeValidator, dynaku
 	return warnIfDeprecatedIsUsed(dynakube, dynatracev1beta1.AnnotationFeatureMetadataEnrichment, dynatracev1beta1.AnnotationFeatureDisableMetadataEnrichment)
 }
 
+func deprecatedFeatureFlag(_ *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
+	warnings := ""
+	for _, ff := range getDeprecatedFeatureFlags() {
+		warnings += warnIfDeprecatedFFisUsed(dynakube, ff)
```

Comment:
did you make sure this shows up correctly, if multiple feature flags are deprecated?

Individual warning messages can't be above 256 characters until they are truncated, see [kubernetes documentation on webhook response](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#response). This could be optimized a bit, by not writing the whole warning message every time, but only the actual feature flag that changed: "Some feature flags are deprecated and will be removed in the future: feature-flag1, feature-flag2, ...". If the truncation is a problem, we could change the message to "feature-flag1, feature-flag2 + 3 more" so it always fits into the message.

Reply:
I did it now in a way that follows your suggestion as I think you are right regarding optimization. Please have a look and give me feedback whether you like this or not.

Reply:
BTW: it worked before but the warning messages were bloated if you used various deprecated feature flags, which is why I tend to your proposed way.

Reply:
FYI: with the approach I introduced in commit https://github.com/Dynatrace/dynatrace-operator/pull/1965/commits/f3475d56283ffb89cc6286b6c3a8c5d48d5e3534 there is no warning at all unfortunately. Working on it

Reply:
With [29279eb](https://github.com/Dynatrace/dynatrace-operator/pull/1965/commits/29279ebd2314cbe33deb61f620f7cde02d863f56) I managed to output the warning message and there seems to be no problem with truncation: 
![Screenshot from 2023-07-24 17-09-59](https://github.com/Dynatrace/dynatrace-operator/assets/62391739/dd3b3177-b816-4cf7-8032-541d8891365a)